pkg/sippyserver/server.go

```diff
@@ -505,17 +545,11 @@ func (s *Server) jsonReleasesReport(w http.ResponseWriter, req *http.Request) {
 	api.RespondWithJSON(http.StatusOK, w, response)
 }
 
-func (s *Server) jsonReleasesReportFromDB(w http.ResponseWriter, _ *http.Request) {
-	type jsonResponse struct {
-		Releases    []string  `json:"releases"`
-		LastUpdated time.Time `json:"last_updated"`
-	}
-	type Release struct {
-		Release string
-	}
-
-	response := jsonResponse{}
+type Release struct {
+	Release string
+}
 
+func (s *Server) releasesFromDB() []Release {
```

Comment:
Could this func be moved to pkg/db/query/release_queries.go (new file)

Reply:
yep